handler/core/src/main/java/io/knotx/fragments/task/TasksEventsWrapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.task;
+
+import com.google.common.collect.Streams;
+import io.knotx.fragments.engine.FragmentEvent;
+import io.knotx.fragments.engine.Task;
+import io.knotx.fragments.task.factory.node.NodeWithMetadata;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+public class TasksEventsWrapper {

```

Comment:
It looks very similar to `TaskEventWrapper` maybe we can combine these classes in some kind of composite?